src/main/java/org/jabref/gui/customentrytypes/CustomizeEntryTypeDialogView.java

```diff
@@ -128,16 +149,88 @@ private void setupTable() {
                 .install(fieldTypeActionColumn);
 
         viewModel.newFieldToAddProperty().bindBidirectional(addNewField.valueProperty());
-        fields.itemsProperty().bindBidirectional(viewModel.fieldsforTypesProperty());
+
+        // Here we would need to select the fields
```

Comment:
I don't get this comment. --> I read it as "We do not need to select the fields and do something different, because of some idea" 😇 

It seems that this is a necessary comment. Could you maybe rephrase it?

Reply:
It was an obsolete comment I forgot to rmeove